OPUNetTransportLayer.cpp

```diff
@@ -1413,9 +1404,11 @@ void OPUNetTransportLayer::CheckSourcePort(Packet &packet, sockaddr_in &from)
 		if ((expectedPort != sourcePort) && (expectedPort != 0))
 		{
 			// Port mismatch. Issue warning
-			logFile << "Packet from player " << sourcePlayerIndex << " (";
-			LogAddress(from);
-			logFile << ") received on unexpected port (" << ntohs(sourcePort) << " instead of " << ntohs(expectedPort) << ")  [PlayerNetId: " << sourcePlayerNetId << "]" << std::endl;
+			Log("Packet from player " + std::to_string(sourcePlayerIndex) + 
+				" (" + FormatAddress(from) + ") received on unexpected port (" + 
+				std::to_string(ntohs(sourcePort)) + " instead of " + 
+				std::to_string(ntohs(expectedPort)) + ")  [PlayerNetId: " + 
+				std::to_string(sourcePlayerNetId) + "]");

```

Comment:
This can eventually become `FormatPlayerNetID`. I was likely too lazy when I wrote this to break up the line and make a separate call to output a formatted value. The new methods are much more convenient to use.

Reply:
I'll push an issue for future incorporation